Distribution/Server/Features/PackageContents.hs

```diff
@@ -214,6 +215,9 @@ renderMarkdown = XHtml.primHtml . Blaze.renderHtml
         , Markdown.debug              = False
         }
 
+convertNewLine :: C8.ByteString -> C8.ByteString
+convertNewLine = C8.concat . C8.split '\r'
```

Comment:
Ah yes, you're right. Sorry, missed this. I'll update this now

A style question: Is it better to now do away with the `convertNewLine` function and use `C8.filter` directly in `renderMarkdown`?